urllib3/response.py

```diff
@@ -422,6 +422,18 @@ def _handle_chunk(self, amt):
             self.chunk_left = None
         return returned_chunk
 
+    @staticmethod
```

Comment:
Why staticmethod? Seems to make sense for it to be a regular instance method and skip passing response in.